client/src/chat/components/ChatScreen.tsx

```diff
@@ -378,6 +415,16 @@ function CreateChatMessage() {
 				/>
 				<BsSendFill onClick={addMessage} css={chatScreenCSS.sendButton} />
 			</div>
+			<div css={chatScreenCSS.emojiPickerIconContainer} onClick={handleShowEmojiPicker}>
+				<BsFillEmojiSunglassesFill />
+			</div>
+			<div css={chatScreenCSS.emojiPickerContainer} ref={ref}>
+				{
+					showEmojiPicker && (
+						<Picker data={data} onEmojiSelect={addEmoji} css={chatScreenCSS.emojiPicker}/>
```

Comment:
Change to `onEmojiSelect={(e)=> addEmoji(e.native)}`, and change addEmoji to take a string, I think would be a bit easier to follow (not a big difference, but it can add up)